scripts.py

```diff
@@ -60,6 +60,10 @@
     PatronInputScript,
     RunMonitorScript,
 )
+# TODO: This class used to be defined in this file and was moved into core.
+# The import statement can be removed once we're sure all references to it
+# have been changed.
+from core.scripts import UpdateSearchIndexScript
```

Comment:
I can't think of any reason something would be referencing this, other than the script file you already changed or a test.